google/cloud/firestore_v1/document.py

```diff
@@ -513,8 +514,8 @@ def on_snapshot(self, callback):
             collection_ref = db.collection(u'users')
 
             def on_snapshot(document_snapshot, changes, read_time):
-                doc = document_snapshot
-                print(u'{} => {}'.format(doc.id, doc.to_dict()))
+                for doc in document_snapshot:
+                    print(u'{} => {}'.format(doc.id, doc.to_dict()))
```

Comment:
I'd make it consistent with the changes to the `collection.py`:
```
            def on_snapshot(docs, changes, read_time):
                for doc in docs:
                    print(u'{} => {}'.format(doc.id, doc.to_dict()))
```
...or vice versa.